app/views/api/v1/orders/show.json.jbuilder

```diff
@@ -0,0 +1,15 @@
+json.instance do
+  json.user @order.user_id
+  json.status @order.status
+
+  if @order.menu_items
+    json.items @order.menu_items do |item|
+      json.id item.id
+      json.item item.name
+      json.price item.price.to_f
+    end
+  else
+    json.items '"no items"'
+  end
+
+end
```

Comment:
Yes, well. In this case the api is returning order object data and menu_items objects data. For me it feels like the most logical format. I also feel that wrapping this data in envelopes seems like good practice and enhances readability. I try (perhaps wrongly) to follow the standards outlined in http://jsonapi.org/